modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase

```diff
@@ -253,12 +253,35 @@ definition {
 	}
 
 	@description = "COMMERCE-6157. As a buyer, I want to see the barred list price and a sale price of a product, if a promotion is set"
-	@ignore = "true"
 	@priority = "3"
 	test ViewProductCardPriceWithPromotion {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6157 ViewProductCardPriceWithPromotion pending implementation
+		CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+		ApplicationsMenu.gotoSite(site = "Minium");
+
+		CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Account Management");
+
+		CommerceAcceleratorsInitializer.createNewAccountViaFront(
+			accountName = "Commerce Account",
+			adminEmailAddress = "test@liferay.com");
+
+		Click(
+			key_navUnderline = "Members",
+			locator1 = "NavUnderline#NAV_UNDERLINE_LINK");
+
+		CommerceAcceleratorsInitializer.inviteNewMemberWithRoleToAccount(
+			emailAddress = "buyer@liferay.com",
+			role = "Buyer");
+
+		ApplicationsMenu.gotoSite(site = "Minium");
+
+		CommerceAcceleratorsInitializer.changeProductsSorting(sortingType = "Name Ascending");
+
+		AssertElementPresent(
+			key_productName = "Brake Pads",
+			locator1 = "CommerceAccelerators#PRODUCT_DETAILS_VALUE_INACTIVE_PROMO_PRICE");
```

Comment:
As for the comment above, please update `CommerceAccelerators#PRODUCT_DETAILS_VALUE_INACTIVE_PROMO_PRICE` locator name.